package.json

```diff
@@ -34,7 +35,7 @@
     "file-loader": "1.1.11",
     "identity-obj-proxy": "3.0.0",
     "jest": "23.4.1",
-    "jest-cli": "23.4.1",
+    "jest-cli": "^23.6.0",
```

Comment:
why'd this get bumped?

Reply:
There were warnings highlighted when I was installing and upgrading the other dependencies, I'll knock it back down if you don't want it bumping for now